diff -Naur avahi-0.8/avahi-libevent.pc.in avahi-0.8.tpg/avahi-libevent.pc.in
--- avahi-0.8/avahi-libevent.pc.in	2018-09-14 05:31:35.000000000 +0000
+++ avahi-0.8.tpg/avahi-libevent.pc.in	2020-02-20 15:13:25.347025581 +0000
@@ -6,6 +6,6 @@
 Name: avahi-libevent
 Description: Avahi Multicast DNS Responder (libevent Support)
 Version: @PACKAGE_VERSION@
-Requires: libevent-2.1.5
+Requires: libevent >= 2.1.5
 Libs: -L${libdir} -lavahi-libevent
 Cflags: -D_REENTRANT -I${includedir}
